src/main/resources/templates/db_scripts/schema.json

```diff
@@ -860,10 +854,6 @@
         }
       ],
       "fullTextIndex": [
-        {
```

Comment:
@julianladisch This index needs to be maintained because when creating a new request, the query 
`SELECT jsonb,id FROM <tenantID>_mod_inventory_storage.item WHERE get_tsvector(f_unaccent(item.jsonb->>?)) @@ tsquery_phrase(f_unaccent(?)) LIMIT ? OFFSET ?`
is used. Without this index creating requests takes minutes in a large DB.

Reply:
See UIREQ-639 "Use == not = for item barcode search": https://issues.folio.org/browse/UIREQ-639